Apps/Playground/node_modules/@babylonjs/react-native/android/build.gradle

```diff
@@ -14,7 +14,7 @@ def DEFAULT_COMPILE_SDK_VERSION = 28
 def DEFAULT_BUILD_TOOLS_VERSION = '28.0.3'
 def DEFAULT_MIN_SDK_VERSION = 18
 def DEFAULT_TARGET_SDK_VERSION = 28
-def DEFAULT_NDK_VERSION = '21.0.6113669'
+def DEFAULT_NDK_VERSION = '21.3.6528147'
```

Comment:
Just curious, any reason we needed to update the NDK version?

Reply:
Ah nevermind I reread the description, this is what is preinstalled on the Github build machines.  This version seems relatively new, are we going to need to regularly update the NDK version?

Reply:
From the PR description :) *"Update the build.gradle files to switch to a different version of the NDK which is pre-installed on GitHub build agents"*

Reply:
Yeah I saw that, I was also wondering if this version is a moving target, are we going to need to periodically keep track and update this to keep our PR builds working?

Reply:
I don't know. I tried not specifying it, but then it just picked some default that is not installed on my machine or on the build agents. :(
As far as I can tell, there is not a way to say "use latest" or anything like that.